### ISSUE TITLE
Integration 'schluter' not found with Hassio v0.103

### ISSUE THREAD

**Issue by Jaharmi** [clumsyCoder00/schluter-python#1]
Using Hassio with Home Assistant 0.103.0, I am unable to verify my configuration when I try to add the schluter component.

        Platform error climate.schluter - Integration 'schluter' not found.

It sounds like there may have been a breaking change in the HA v0.89 or v0.90 release that could be related to this. I’m not sure what to do to fix this, myself.

**Comment by clumsyCoder00** [comment]
Thanks for the comment. I plan on having a fix for this issue by the end of the week and issue an update in this thread. Happy to hear that someone else can find the integration useful!

**Comment by clumsyCoder00** [comment]
I've updated this integration. Please let me know if this fixes your issue, and I can further assist in troubleshooting or close this issue. Thanks!

**Comment by Jaharmi** [comment]
I had to rebuild my Home Assistant / Hassio setup due to a problem with the Git Pull add-on. I’m now using Home Assistant 0.103.5.

I’ve tried adding the `schluter` directory under `custom_components` and then into my `configuration.yaml` file, with the login information in my `secrets.yaml` file. Initially, it wasn’t working for me and threw a notification about the add-on.

Now it’s working after I worked on rebuilding other parts of my config. My `configuration.yaml` includes:

    climate:
      - platform: schluter
      email: !secret schluter_userid
      password: !secret schluter_password

Also, I’m not sure what it takes, but I couldn’t get the repo to work with HACS.
